docs/can-guides/commitment/recipes/playlist-editor/3-search.js

```diff
@@ -31,22 +31,22 @@ var PlaylistVM = can.DefineMap.extend("PlaylistVM", {
   get searchResultsPromise() {
     if (this.searchQuery.length > 2) {
 
-      var results = gapi.client.youtube.search.list({
+      const results = gapi.client.youtube.search.list({
           q: this.searchQuery,
-          part: 'snippet',
-          type: 'video'
-        }).then(function(response){
+          part: "snippet",
+          type: "video"
+        }).then(function(response) {
         console.log(response.result.items);
```

Comment:
I know this isn't new, but do we need this `console.log`?

Reply:
+1 on removing this; not sure if @justinbmeyer intentionally left those in because they’re useful for other people going through the guide.

Reply:
For the sake of time (not updating the highlighting for all the examples), I’ve changed these to `console.info("Search results:", response.result.items)`